js/constants/config.js

```diff
@@ -7,7 +7,7 @@
 // https://vault-staging.brave.com for a dev build
 // http://localhost:3000 for production
 const vaultHost = process.env.VAULT_HOST || 'https://vault-staging.brave.com'
-const adHost = process.env.AD_HOST || '//cdn.brave.com'
+const adHost = process.env.AD_HOST || '//oip.brave.com'
```

Comment:
Any reason not to just change this to https instead of a protocol-relative URL? Otherwise a passive MITM sees information about a user's ad preferences that otherwise wouldn't be exposed.